libfuse/platform_flags_osx.go

```diff
@@ -27,5 +28,8 @@ func AddPlatformFlags(flags *flag.FlagSet) *PlatformParams {
 	var params PlatformParams
 	flags.BoolVar(&params.UseSystemFuse, "use-system-fuse", false,
 		"Use the system OSXFUSE instead of keybase's OSXFUSE")
+	flags.BoolVar(&params.UseLocal, "local-experimental", false,
+		"Use 'local' mount option"+
+			" and enable other hacky stuff for testing macOS apps.")
```

Comment:
Maybe explain that for now, with this flag you can only work within your private home dir?